doc/source/deframer_packet_processor.rst

```diff
@@ -0,0 +1,122 @@
+AIT Deframer/Packet Processor
+=============================
+
+This is a guide for AIT's base implementation for the Deframer and Packet Processor.
+
+The processor receives transfer frames (AOS or TM), extracts CCSDS packets from those frames, and then organizes and emits those packets downstream.
+
+The base implementation handles AOS tansfer frames containing partials and will create full packets from partials when possible.
+AIT does not currently support partials from TMTransferFrames, so they are not handled in this implementation either.
+
+
+Configuration
+^^^^^^^^^^^^^
+
+The necessary configuration parameters can be added to the **config.yaml**.
+Optionally, many of them can also be passed in at runtime for greater flexibility.
+Below is an example configuration which can be used as a reference, in addition to the default configuration available in the repository.
+
+
+.. code-block:: yaml
+
+    dsn:
+        proc:
+            timer_poll: 15                # How long to poll for new telemetry, seconds
+            cleanup_poll: 300             # How often to check for cleanup, seconds
+            max_gap: 100                  # If 0, do not allow gaps in packet downstream
+            packet_output_port: 3076      # The UDP port used for emitting CCSDS packets
+        sle:
+            downlink_frame_type: TMTransFrame  # or AOSTransFrame
+            frame_output_port: 3726    # The incoming UDP port for transfer frames
+
+     Note: If max_gap is set to 0, then any gaps in incoming packets with the same APID will necessarily prevent emission of packets until the gap is resolved.
+     As such, it would only be recommended for use during playback of complete telemetry.
+
+    Any of the above base keywords can also be set using a kwargs that is passed into the Processor.
+
+Running the Processor
+^^^^^^^^^^^^^^^^^^^^^
+
+The processor can be run as a command line:
+
+::
+
+    python ait/dsn/proc/deframe_packet_processor.py
```

Comment:
Update to `bin/` script call per comment above about moving `__main__` execution code out to a separate bin script for consistency